### PR TITLE
Add a note about Operators.

### PR DESCRIPTION
**Pull request by yhontyk**
Branch 4.6 and 4.7

### CODE REVIEW

**Review by dharmit — COMMENTED** [review]
This change looks good to me from odo side.

File: cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc [hunk 1]
```diff
@@ -7,10 +7,14 @@ toc::[]
...
+[NOTE]
+====
+You cannot install Operators with `{odo-title}` on your cluster. To install Operators, contact your cluster administrator or see xref:../../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace-prereqs[Installing Operators in your namespace]
+====
 
-== Prerequisites
+To create services from an Operator, you must ensure that the Operator has valid values defined in its `metadata` to start the requested service. `{odo-title}` uses the `metadata.annotations.alm-examples` YAML file of an Operator to start the service. If this YAML has placeholder values or sample values, a service cannot start. You can modify the YAML file and start the service with the modified values. To learn how to modify YAML files and start services from it, go to xref:../../cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc#creating-services-from-yaml-files_creating-instances-of-services-managed-by-operators[Creating services from YAML files].
```
Comment (pneedle-rh) [inline cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc:15]: You could consider swapping the order of the first three sentences. I suggest introducing the `metadata.annotations.alm-examples` YAML file, then introducing the requirements for the file. For example:

"`{odo-title}` uses the `metadata.annotations.alm-examples` YAML file of an Operator to start a service from an Operator. A service cannot start if this YAML file has placeholder or sample values. To start a service, you must ensure that the `metadata` section of the YAML has valid values."

Comment (pneedle-rh) [reply cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc:15]: I suggest changing "To learn how to modify YAML files and start services from it" to "To learn how to modify YAML files and start services from them".

**Review by pneedle-rh — COMMENTED** [review]
@boczkowska I have now completed my review. Please let me know if you have any questions.

**Comment by pneedle-rh** [comment]
@boczkowska please let me know if you have any questions about my previous review comments or about what is required to rebase this pull request. Thanks.

**Pull request closed without merging**